src/sentry/integrations/utils/code_mapping.py

```diff
@@ -210,18 +232,27 @@ def list_file_matches(self, frame_filename: FrameFilename) -> List[Dict[str, str
         return file_matches
 
     def _get_code_mapping_source_path(self, src_file: str, frame_filename: FrameFilename) -> str:
-        """Generate the source path of a code mapping
-        e.g. src/sentry/identity/oauth2.py (sentry/identity/oauth2.py) -> src/sentry
-        e.g. src/sentry/wsgi.py (sentry/wsgi.py) -> src/sentry
-        e.g. ssl.py -> raise NotImplementedError
-        """
-        if frame_filename.dir_path != "":
-            source_path = src_file.rsplit(frame_filename.dir_path)[0].rstrip("/")
-            return f"{source_path}/"
-        elif frame_filename.root != "":
-            return src_file.rsplit(frame_filename.file_name)[0]
+        """Generate the source code root for a code mapping"""
+        source_code_root = None
+        if frame_filename.frame_type() == "packaged":
+            if frame_filename.dir_path != "":
```

Comment:
This block has been right shifted. If you use "Hide whitespace" it is easier to see any differences.
<img width="174" alt="image" src="https://user-images.githubusercontent.com/44410/205089314-a10b965a-cb5e-4071-a766-f8ce525ab0f9.png">